macOS/LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/macOS/Utils/DataBrokerProtectionBundleExtension.swift

```diff
@@ -21,13 +21,14 @@ import Foundation
 extension UserDefaults {
     static let dbp = UserDefaults(suiteName: Bundle.main.dbpAppGroup)!
     static let config = UserDefaults(suiteName: Bundle.main.configAppGroup)!
+    static let netP = UserDefaults(suiteName: Bundle.main.netPAppGroup)!
 }
 
 extension Bundle {
 
     var dbpAppGroup: String {
         guard let appGroup = object(forInfoDictionaryKey: Bundle.dbpAppGroupName) as? String else {
-            fatalError("Info.plist is missing \(appGroupName)")
+            fatalError("Info.plist is missing \(Bundle.dbpAppGroupName)")
```

Comment:
This is fine but just noting that `Bundle.` isn't necessary here.

Reply:
`dbpAppGroupName` is a static var so we need `Bundle.`